dev_config/python/.pre-commit-config.yaml

```diff
@@ -1,17 +1,19 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v4.5.0
     hooks:
       - id: trailing-whitespace
       - id: end-of-file-fixer
       - id: check-yaml
       - id: debug-statements
-      - id: double-quote-string-fixer
 
-  - repo: https://github.com/hhatto/autopep8
```

Comment:
Need to revert this change. We could keep autopep8 as long as it doesn’t fight with others.